PennMobile/About/Privacy/PrivacyPreference.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PrivacyPreference.swift
+//  PennMobile
+//
+//  Created by Dominic Holmes on 12/27/19.
+//  Copyright © 2019 PennLabs. All rights reserved.
+//
+
+import Foundation
+
+typealias PrivacyPreferences = Dictionary<PrivacyOption, Bool>
+
+enum PrivacyOption: String, Codable, CaseIterable {
```

Comment:
Are you using CaseIterable anywhere? It doesn't seem so. If not, we should delete it.